addons/website/tests/test_qweb.py

```diff
@@ -107,3 +107,86 @@ def test_process_att_url_crap(self):
             match.reset_calls()
             self._test_att('/x?y#z', {'href': '/x?y#z'})
             match.assert_called_with('/x', method='POST', query_args='y')
+
+class TestQwebDataSnippet(TransactionCase):
+    def setUp(self):
+        super().setUp()
```

Comment:
I think you should use `setUpClass` instead of `setUp` (see examples), otherwise this setup is done at the start of every sub-test. You can test that by launching all class tests with the command `--test-tags=:TestQwebDataSnippet` if you don't know (see the `:` instead of `.` to mention a class instead of a subtest).